package.json

```diff
@@ -10,5 +12,8 @@
     "co": "4.5.x",
     "data.task": "3.0.x",
     "ramda": "0.15.x"
+  },
+  "devDependencies": {
+    "babel": "^5.8.9"
```

Comment:
I'm assuming that the project is following SemVer and that any `5.x.x` version will be compatible. If you'd prefer to be more specific, let's go with `"5.8.x"` to match the format used for the other dependencies.